tests/integration/test_integration.py

```diff
@@ -15,7 +15,7 @@
 METADATA = yaml.safe_load(Path("./charmcraft.yaml").read_text())
 APP_NAME = METADATA["name"]
 DB_CHARM_NAME = "mongodb-k8s"
-DB_CHARM_CHANNEL = "6/beta"
+DB_CHARM_CHANNEL = "6/stable"
```

Comment:
Why are we changing the risk on DB charm? In the terraform modules we are still using `6/beta`: this would lead to possible incosistent behaviours between test results and actual SD-Core deployment.

Reply:
I used stable here because of an issue in the beta version of the charm. We would be getting this ` hook failed: "upgrade-version-a-relation-created"` consistently without this change.

Additionally, yes we will want to use `6/stable` everywhere. This wasn't possible for a while since the charm's highest risk was beta. So yes, we should bump mongodb to `6/stable` in the end-to-end tests and other charms as well.

Ref here:
- https://github.com/canonical/sdcore-amf-k8s-operator/actions/runs/11541296401/job/32123966461